src/actions/account.js

```diff
@@ -41,6 +43,17 @@ export function handleRefreshAccount(wallet, history) {
          })
          .catch(e => {
             console.log(e)
+
+            if (e.message && e.message.indexOf('is not valid') !== -1) {
```

Comment:
pull 'is not valid' into a constant somewhere?

Reply:
or better yet this function might be good as a util function

Reply:
proper way to fix this would be to haver error codes returned by `nearcore`. This works as a temporary kludge as is.